ReactiveUsersMicroservice/src/main/java/Cloud2024/ReactiveUsersMicroservice/Infrastructure/UserMapper.java

```diff
@@ -22,6 +22,7 @@ public ModelMapper modelMapper(){
                     mapper.map(src -> generateRandomAsterisks(), UserBoundary::setPassword);
                     mapper.map(UserEntity::getFirstName, (dest, v) -> dest.getName().setFirst((String) v));
                     mapper.map(UserEntity::getLastName, (dest, v) -> dest.getName().setLast((String) v));
+//                    mapper.map(src -> src.getDeptId(), UserBoundary::setDepartment);
```

Comment:
@niv-e having trouble with the mapper, in UserBoundary there is a Department field that is supposed to be converted to deptId in the UserEntity. i am having trouble implpementing it in the mapper.